src/main/java/com/minecolonies/coremod/entity/ai/mobs/AbstractEntityBarbarian.java

```diff
@@ -0,0 +1,140 @@
+package com.minecolonies.coremod.entity.ai.mobs;
+
+import com.minecolonies.api.configuration.Configurations;
+import com.minecolonies.coremod.items.ItemChiefSword;
+import com.minecolonies.coremod.sounds.BarbarianSounds;
+import com.minecolonies.coremod.util.BarbarianSpawnUtils;
+import com.minecolonies.coremod.util.BarbarianUtils;
+import net.minecraft.entity.EntityLivingBase;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'net.minecraft.entity.EntityLivingBase'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)